views/dataoutputs.erb

```diff
@@ -193,6 +193,7 @@
                ['Riemann', 'riemann.png', 'https://github.com/wkf/fluent-plugin-riemann'],
                ['Sentry', 'getsentry.png', 'https://github.com/y-ken/fluent-plugin-sentry'],
                ['New Relic', 'new-relic.png', 'https://github.com/newrelic/newrelic-fluentd-output'],
+               ['Observe', 'observe.png', 'https://docs.observeinc.com/en/latest/content/data-ingestion/forwarders/fluentbit.html'],
```

Comment:
Here is Fluentd's web site, not Fluent Bit.
So I think https://docs.observeinc.com/en/latest/content/data-ingestion/forwarders/fluentd.html is appropriate than it.

Reply:
Thank you @ashie.  This makes sense.  I actually created a PR for that site as well:  https://github.com/fluent/fluent-bit-docs/pull/804

Once that PR is evaluated I'll close this PR.  Thanks for all your help.